zetaclient/config/config.go

```diff
@@ -76,19 +83,116 @@ func Load(path string) (Config, error) {
 	// fields sanitization
 	cfg.TssPath = GetPath(cfg.TssPath)
 	cfg.PreParamsPath = GetPath(cfg.PreParamsPath)
-	cfg.ZetaCoreHome = path
-
-	// load compliance config
-	LoadComplianceConfig(cfg)
+	cfg.ZetaCoreHome = basePath
 
 	return cfg, nil
 }
 
-// LoadComplianceConfig loads compliance data (restricted addresses) from config
-func LoadComplianceConfig(cfg Config) {
+// SetRestrictedAddressesFromConfig loads compliance data (restricted addresses) from config.
+func SetRestrictedAddressesFromConfig(cfg Config) {
 	restrictedAddressBook = cfg.GetRestrictedAddressBook()
 }
 
+func getRestrictedAddressAbsPath(basePath string) (string, error) {
+	file := filepath.Join(basePath, folder, restrictedAddressesPath)
+	file, err := filepath.Abs(file)
+	if err != nil {
+		return "", fmt.Errorf("abs %s: %w", file, err)
```

Comment:
To make error handling consistent, I think `errors.Wrap/Wrapf` is a good choice. Same for newly added code below.

Reply:
Updated